optuna/integration/fastai.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import print_function
+
+import optuna
+from optuna import type_checking
+
+if type_checking.TYPE_CHECKING:
+    from typing import Any  # NOQA
+
+try:
+    from fastai.basic_train import Learner  # NOQA
+    from fastai.callbacks import TrackerCallback
+    _available = True
+except ImportError as e:
+    _import_error = e
+    _available = False
+    Callback = object
```

Comment:
Is this a typo of `TrackerCallback`?
```suggestion
    TrackerCallback = object
```

Reply:
By fixing this typo, it seems [the condition](https://github.com/pfnet/optuna/pull/585/files#diff-4f60853ac0648ba2dfecfa69ee7d64b5R45) can be removed and it will resolve [the problem of `checks` job](https://github.com/pfnet/optuna/pull/585#issuecomment-543500752).

Reply:
Thanks.

I initially implemented this callback on top of `Callback` before finding `TrackerCallback`.